pallets/permissions/src/lib.rs

```diff
@@ -76,8 +75,8 @@ pub enum Scope {
     Unlimited,
 }
 
-pub const MINT: PermissionId = 2;
-pub const BURN: PermissionId = 3;
+pub const MINT: PermissionId = common::MINT;
+pub const BURN: PermissionId = common::BURN;
 pub const SLASH: PermissionId = 4;
 pub const INIT_DEX: PermissionId = 5;
 pub const MANAGE_DEX: PermissionId = 6;
```

Comment:
Don't split permission ids into several files. Select the one place for all permission ids.